### DISCUSSION TITLE
How to install TinyMQTT on a ESP32 with Tasmota ?

### DISCUSSION THREAD

**Discussion by roesl** [hsaturn/TinyMqtt#51] [Ideas]
Is it possible, to add TinyMQTT to Tasmota ?

**Comment by hsaturn** [comment]
Sorry Roesl, I do not often check discussions. (And am not receiving notification for them).

What do you mean exatly by integrating TinyMqtt to tasmotta ?

**Comment by roesl** [comment]
Hi, I would like to use an esp32 with tasmota also as a MQTT server, so there is no need of an external server. Regards Georg --Diese Nachricht wurde von meinem Android Mobiltelefon mit 1&1 Mail gesendet.Am 28.02.23, 11:00 schrieb hsaturn ***@***.***>:

   
  Sorry Roesl, I do not often check discussions. (And am not receiving notification for them). 
  What do you mean exatly by integrating TinyMqtt to tasmotta ?

**Comment by hsaturn** [comment]
Hello

This should be possible. TinyMqtt code is light, less than 280Kbytes. So it can exist beside Tasmotta or be integrated with it as a broker in a ESP32 device that has enough ROM and RAM to do this.

I even guess that this could be done with an ESP8266 when using a reduced binary ~1Mb, don't know about ram usage.
As I don't know much about Tasmotta (only flashed it once on a device one time), I cannot say more.

Best regards
